src/database/type-transforms.tsx

```diff
@@ -81,3 +87,9 @@ export const buildScenario = (
 
   return scenario;
 };
+
+export const buildUser = (document: firebase.firestore.DocumentData): User => {
+  const data = document.data();
+
+  return { ...pick(data, ["name", "email"]), id: document.id };
```

Comment:
I might be making an ultra-obvious mistake, but shouldn't we also be picking `"auth0Id"`? Why is that field not on the user type, more generally?

Reply:
I left it out because a) we don't need it and b) we don't want it to leak. In this situation users are only looking up themselves, but once they start looking up others (i.e. when we implement the actual sharing feature) we don't want them having access to everyone's auth0 credential. 

We do need it to be in the database, which is why it's required for new document creation, but other than that we shouldn't be passing it around.